man/bootctl.xml

```diff
@@ -377,6 +377,12 @@
         <listitem><para>Install binaries for all supported EFI architectures (this implies <option>--no-variables</option>).</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--efi-label=</option></term>
```

Comment:
hmm, that is a bit generic a name. maybe `--efi-boot-option-description="`?

This uses the EFI spec terminology, where these things are called "boot options" and they have a `Description` field, for the human readable string.

There's really no need to be terse here, given this should normally not be an option you have to use daily.

Reply:
Done. +1 on the suggestion on following the UEFI spec here. Initially got "label" from efibootmgr `--label` flag.